makeui.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/sh
-
+=======
 set -x # Print command traces before executing command
 set -e # Exit immediately if a simple command exits with a non-zero status.
 
-pyuic5 pytripgui/view/main_window.ui -o pytripgui/view/main_window.py
-pyuic5 pytripgui/view/trip_config.ui -o pytripgui/view/trip_config.py
+pyuic5 pytripgui/view/main_window.ui > pytripgui/view/gen/main_window.py
```

Comment:
Why do you reverted my change to use "-o" switch ?

Reply:
As you changed the path, could you also please update it in several other places:
https://github.com/pytrip/pytripgui/blob/d449908ae631bda7abc51dcc186a86ab9e207c1d/tox.ini#L21
Can you try also to look for the "main_window.py" in other files ?

Later I will centralise this locations.

This should also fix tests